src/views/product-integration/component-view/components/variable-group-list/index.tsx

```diff
@@ -35,32 +39,52 @@ export function VariableGroupList({
 				return (
 					<li
 						key={variable.key}
-						className={classNames({
+						className={classNames(s.variableGroupListItem, {
 							[s.highlight]: variable.highlight,
 						})}
 					>
-						<code className={s.key}>
-							<strong>{variable.key}</strong>
-						</code>
-						<code className={s.type}>{variable.type}</code>
-						{typeof variable.description !== 'undefined' && (
-							<div>
+						{isNested ? (
+							<div className={s.arrowIcon}>
+								<IconCornerDownRight16 />
+							</div>
+						) : (
+							<></>
+						)}
```

Comment:
Nit: We can use [logical &&](https://reactjs.org/docs/conditional-rendering.html#inline-if-with-logical--operator) instead of ternary+fragment for conditional rendering

```suggestion
						{isNested && (
							<div className={s.arrowIcon}>
								<IconCornerDownRight16 />
							</div>
						)}
```

Reply:
Ah!  cc @thiskevinwang looks like this article is suggesting ternary over &&, whereas you suggested logical && over ternary.

I see how you can get the job done with either; curious if there is an explicit preference that we've agreed upon on the team?  @kendallstrautman might be a good thread to open up for discussion in the channel (will leave that to you to do though if you want / not).

Reply:
I'm not super tied to either approach!

I've historically reached for both `x && y` as well as `x ? y : z` ...(:homer:)
- The article sounds like it advocates for ternary for mitigating a small class of bugs.

Reply:
(Feel free to disregard my "nits")

Reply:
I don't have a strong opinion either (but I do have an opinion that *we* should pick what the default favored approach should be)

Reply:
Going to resolve this for _this PR_ -- maybe let's bring this one to the team to see if we can land on a preferred style.